pkgs/applications/networking/browsers/mozilla-plugins/esteidfirefoxplugin/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, gtk2, openssl, pcsclite, pkgconfig, opensc }:
+
+stdenv.mkDerivation rec {
+  version = "3.12.1.1142";
+  name = "esteidfirefoxplugin-${version}";
+
+  src = fetchurl {
+    url = "https://installer.id.ee/media/ubuntu/pool/main/e/esteidfirefoxplugin/esteidfirefoxplugin_3.12.1.1142.orig.tar.xz";
+    sha256 = "0y7759x1xr00p5r3c5wpllcqqnnxh2zi74cmy4m9m690z3ywn0fx";
+  };
+
+  unpackPhase = ''
```

Comment:
Hm, I tried it with `qdigidoc`, seems to work. I'd think that simply `unpackPhase = ''tar xf $src''` would work, too. Anyway, not terribly important ...